pkg/cloudevents/event_interface.go

```diff
@@ -66,4 +66,7 @@ type EventWriter interface {
 
 	// SetExtension performs event.Context.SetExtension.
 	SetExtension(string, interface{})
+
```

Comment:
AI: alphabetize the Set operations at some point.

Reply:
I have been attempting to keep them in the order the spec introduces them: https://github.com/cloudevents/spec/blob/master/spec.md